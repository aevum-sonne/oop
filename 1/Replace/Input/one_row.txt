The Smiths signed to the independent record label Rough Trade Records.
