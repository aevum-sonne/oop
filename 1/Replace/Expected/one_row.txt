The Smiths signed to 123 independent record label Rough Trade Records.
